private/bufpkg/bufmodule/bufmodule.go

```diff
@@ -413,6 +421,7 @@ func ModuleDigestB3(ctx context.Context, module Module) (string, error) {
 			return "", err
 		}
 	}
+	// TODO: Should we include module.DocumentationPath here?
```

Comment:
Should the documentation path be included in the module digest?

Reply:
I think so -- I think it is desirable to have a push occur when the documentation path changes, so that should result in a digest change.

Reply:
hm thinking if it would break anyone. I guess it would result in a digest change if you had an existing `buf.md` file but didn't change anything. Could that be a problem?

Reply:
Continuing in the other [thread](https://github.com/bufbuild/buf/pull/2027#discussion_r1180842068) below.